packages/storage-plus/src/de.rs

```diff
@@ -0,0 +1,119 @@
+use crate::keys::{IntKey, TimestampKey};
+use cosmwasm_std::{Addr, StdError, StdResult};
+use std::array::TryFromSliceError;
+use std::convert::TryInto;
+
+pub trait KeyDeserialize {
```

Comment:
I wonder if it should return an error (eg. for cases where slice is of wrong length, so there is obviously something wrong). Obviously panicking is wrong, but I am not sure about silently accepting illformed data.

Reply:
It will error, through `try_from`.